components/ui/CustomText.tsx

```diff
@@ -11,6 +11,7 @@ const typographyStylesMap: Record<SizeVariant, TextStyle> = {
   xl: { fontSize: 28, lineHeight: 36.32 },
   lg: { fontSize: 18, lineHeight: 21.85 },
   base: { fontSize: 16, lineHeight: 19.42 },
+  md: { fontSize: 15, lineHeight: 20 },
```

Comment:
Since the typography is being updated in another [PR](https://github.com/wavvapp/sac-frontend/pull/71) with a different approach, I think it's just better to use customStyles where you need this fontSize for now to avoid possible conflicts. Tehn when the typography is merged we can replace the customStyles